### PR TITLE
Entrega 4 Marcos Garcia Postigo

### PR DESCRIPTION
**Pull request by bayoneta101**

### CODE REVIEW

File: Assignment4/bayoneta101-190375/task06.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/14FtrvhtMq0tkliJKhES4Iu2p1PxH3NBv
+
+**Task 06: Modifying RDF(s)**
+"""
+
+!pip install rdflib
```
Comment (dgarijo) [inline Assignment4/bayoneta101-190375/task06.py]: won't work in python. This is  Notebook magic

File: Assignment4/bayoneta101-190375/task07.py [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+  print(s)
+
+# SPARQL
+q2 = prepareQuery('''
+  SELECT 
+    ?y
+  WHERE { 
+    {?y rdf:type ns:Person.} UNION
+    {?y rdf:type ?x.
+     ?x rdfs:subClassOf ns:Person }
```
Comment (dgarijo) [inline Assignment4/bayoneta101-190375/task07.py]: use * or + to list all subclasses of Person, recursively

File: Assignment4/bayoneta101-190375/task07.py [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+q3 = prepareQuery('''
+  SELECT DISTINCT ?s ?p ?o
+  WHERE{
+  	{ 
+      ?s rdf:type ns:Person.
+      ?s ?o ?p
+    } UNION {
+      ?s rdf:type ?t.
+      ?s ?o ?p.
+      ?t rdfs:subClassOf ns:Person
```
Comment (dgarijo) [inline Assignment4/bayoneta101-190375/task07.py]: same as above

**Review by dgarijo — CHANGES_REQUESTED** [review]

**Comment by bayoneta101** [comment]
Añadido group assigment 1, ahora arreglare el task 6 y 7

**Comment by bayoneta101** [comment]
Branch listo para merge, con task 6,7 y group assigment 1

File: Assignment4/bayoneta101-190375/task07.py [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+
+#SPARQL
+from rdflib.plugins.sparql import prepareQuery
+
+
+q1 = prepareQuery('''
+  SELECT 
+    ?x
+  WHERE { 
+    ?x rdf:type ns:Person.
```
Comment (dgarijo) [inline Assignment4/bayoneta101-190375/task07.py]: This is listing all persons, not subclasses of person

**Review by dgarijo — CHANGES_REQUESTED** [review]
Please fix the pending error.

**Review by dgarijo — APPROVED** [review]

**Pull request merged**